config/hubs/openscapes.cluster.yaml

```diff
@@ -60,18 +60,27 @@ hubs:
                   [
                     "sh",
                     "-c",
-                    "id && chown 1000:1000 /home/jovyan && ls -lhd /home/jovyan",
+                    "id && chown 1000:1000 /home/jovyan /home/jovyan/shared && ls -lhd /home/jovyan",
```

Comment:
Is we chown on a dir that doesn't exist, this fails right?

Q: can/should we assume this to exist?